torch/distributed/_tensor/examples/comm_mode_features_example.py

```diff
@@ -109,6 +131,16 @@ def test_MLP_module_tracing(self) -> None:
         """
         Example code to demonstrate CommModeDebug's module level tracing using a MLP model.
         Prints a table of module level collective tracing information
+
+        Expected Output
+        Global
+        c10d_functional.all_reduce: 1
+        MLPModule
+            c10d_functional.all_reduce: 1
+            MLPModule.net1
+            MLPModule.relu
+            MLPModule.net2
+            c10d_functional.all_reduce: 1
```

Comment:
question: How to read it when collectives and submodules are having the same indentation, i.e. which module does a collective belong, and does the order matter?

Reply:
Since this test covers both forward and backward, will collectives in both forward and backward be captured? If so, is there a way to differentiate them?

Reply:
regarding the indentation, the change in #128721 seems good.

Yes collectives in both forward and backward be captured. I think we can distinguish them in Module Tracker. cc @sinhaanshul